src/app/src/components/Visualizations.jsx

```diff
@@ -114,7 +114,7 @@ const Visualizations = () => {
                             {questions.map(items => (
                                 <Chart
                                     items={items}
-                                    key={items[0].question.qcode}
+                                    key={`${items[0].response.key}-${items[0].response.geo}`}
```

Comment:
It does not appear that Stacked type charts have a `response` key, and this line will generate an error when a question of that type is selected.

```
TypeError: Cannot read property 'key' of undefined
    at Visualizations.jsx:117
```

Reply:
Fixed in ca1cb5e